GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.ts

```diff
@@ -924,6 +924,14 @@ namespace gdjs {
       // HTML5 games on mobile/browsers don't have a way to close their window/page.
     }
 
+    dispose() {
+        this._pixiRenderer?.destroy(true);
+        this._pixiRenderer = null;
+        this._threeRenderer = null;
+        this._gameCanvas = null;
```

Comment:
Remove gameCanvas from the parent element.

Reply:
done